#!/usr/bin/env python
PACKAGE = "franka_interactive_controllers"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("translational_stiffness", double_t, 0, "Cartesian translational stiffness", 400, 0, 600)
gen.add("rotational_stiffness", double_t, 0, "Cartesian rotational stiffness", 60, 0, 70)
gen.add("nullspace_stiffness", double_t, 0, "Stiffness of the joint space nullspace controller (the desired configuration is the one at startup)", 10, 0, 100)
gen.add("activate_tool_compensation", bool_t, 0, "trigger to activate external tool compensation (the desired compensation forces defined in ./config)", True)

exit(gen.generate(PACKAGE, "dynamic_compliance", "compliance_param"))
